hail/python/hail/methods/impex.py

```diff
@@ -1989,9 +1989,10 @@ def import_plink(bed, bim, fam,
            _intervals=nullable(sequenceof(anytype)),
            _filter_intervals=bool,
            _drop_cols=bool,
-           _drop_rows=bool)
+           _drop_rows=bool,
+           _n_partitions=nullable(int))
 def read_matrix_table(path, *, _intervals=None, _filter_intervals=False, _drop_cols=False,
-                      _drop_rows=False) -> MatrixTable:
+                      _drop_rows=False, _n_partitions=None) -> MatrixTable:
```

Comment:
I'm wondering if we made a mistake here with the `_names`. I guess it's to mark instability, but there's no real path to stability as well.

Just a thought.